import_openssl.sh

```diff
@@ -35,9 +35,28 @@ export LANG=C
 export LC_ALL=C
 PERL_EXE="perl -C0"
 
-if [ ! -x ${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/darwin-x86_64/bin/armv7a-linux-androideabi28-clang++ ]; then
-    echo ANDROID_NDK_HOME not set to a valid directory
-    exit 1
+# Declare Android-NDK ABI-Level
+NDK_TOOLCHAIN_ABILVL=28
```

Comment:
I think SDKINT instead of ABILVL is more normal way of expressing this in Android land.